GCRCatalogs/dr_register.py

```diff
@@ -157,7 +158,10 @@ class DrConfigRegister(RootDirManager, DrConfigManager):
         def __init__(self, site_config_path=None, user_config_name=None,
                      dr_root=None, dr_schema=None, dr_site=None,
                      owner_type=None, owner=None):
-            if dr_schema.find("production") > -1:
+            if dr_schema is None:
+                owner_type = "production"
+                owner = "production"
+            elif dr_schema.find("production") > -1:
```

Comment:
Is there a reason that these two conditions are not combined in one if statement? 
```suggestion
            if dr_schema is None or dr_schema.find("production") > -1:
```

Reply:
If `dr_schema` is None the second condition would throw an exception if executed first.   I was being cautious. In a compiled language it's possible that the order of execution could change but, after some googling, I'm more or less convinced that's not a concern for Python. I will accept your suggestion.